src/server/services/wms/qgsmaprendererjobproxy.cpp

```diff
@@ -27,15 +27,11 @@ namespace QgsWms
   QgsMapRendererJobProxy::QgsMapRendererJobProxy(
     bool parallelRendering
     , int maxThreads
-#ifdef HAVE_SERVER_PYTHON_PLUGINS
```

Comment:
@elpaso If HAVE_SERVER_PYTHON_PLUGINS is defined then accessControl is an opaque type and the pointer returned by the serverInterface in set to null: by doing this we can safely remove all conditionals on parameter list (but not those associated to specific code).